src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -65,6 +65,11 @@ public class JwtTokenUtilities
             JwtHeaderParameterNames.Zip
         };
 
+        /// <summary>
+        /// Creates a dictionary from a list of Claim's.
+        /// </summary>
+        /// <param name="claims"> A list of claims.</param>
+        /// <returns> A Dictionary representing claims.</returns>
         internal static Dictionary<string, object> CreateDictionaryFromClaims(IEnumerable<Claim> claims)
```

Comment:
It's preferable to return IDictionary<string, object> than Dictionary<string, object>.